### ISSUE TITLE
Create app category for journaling applications

### ISSUE THREAD

**Issue by phase1geo** [myfreeweb/awesome-gtk#83]

**Comment by loicreynier** [comment]
Which apps would you put in this category?

**Comment by phase1geo** [comment]
Specifically, Journaler (https://github.com/phase1geo/journaler). Journaling is different from note-taking (which is the closest app category listed).

**Comment by loicreynier** [comment]
I believe that "note taking" is sufficiently close to "journaling" for us to wait until we have multiple applications before creating a new category.

**Comment by phase1geo** [comment]
RedNotebook is also a GTK3 journaling/diary application which exists on your page and is also currently listed as a note-taking application.  It would be nice to have a separate category for these types of apps.  They really are different types of apps from note-taking and deserve a category of their own.

**Comment by loicreynier** [comment]
Once #84 is merged, we should indeed add this category.

**Comment by loicreynier** [comment]
Closed in #86.
